docs/contributing/using-an-online-dev-environment.md

```diff
@@ -1,52 +1,52 @@
 
-## About Gitpod
+## Sobre o Gitpod
 
-Gitpod makes contributions on GitHub easier, by allowing automated development environment setups. Instead of writing lengthy
-documentation of what to install and configure and letting every contributor go through this adventure, the setup is automated and reproducible.
+Gitpod deixa as contribuições no Github mais fáceis, pois permite a configuração automatizada de um ambiente de desenvolvimento. Ao invés de escrever uma longa
+documentação de como instalar e configurar deixando todo contribuidor passar por essa aventura, a configuração é automatizada e reproduzível.
 
-Furthermore, Gitpod prebuilds any branch of a repository so that you don't need to wait for installation, cloning and building.
+Além disso, o Gitpod pré-builda qualquer branch de um repositório para que você não precise aguardar por uma instalação, clonando e buildando.
 
-## Get Started
+## Iniciando
 
-To start a fresh dev environment you can prefix the any GitHub URL with `gitpod.io/#`.
+Para iniciar um ambiente de desenvolvimento do zero, você pode prefixar qualquer URL do Github com `gitpod.io/#`.
 
-> Example: https://gitpod.io/#https://github.com/gatsbyjs/gatsby
+> Exemplo: https://gitpod.io/#https://github.com/gatsbyjs/gatsby
 
-The started dev environment will open with a ready-built gatsby core project as well as one built example (gatsbygram).
-Three terminals are started side-by-side running the following processes:
+O ambiente de desenvolvimento irá abrir um projeto core do gatsby já buildado, bem como um exemplo de build (gatsbygram).
+Três terminais serão iniciados lado a lado executando os seguintes processos:
 
 - `yarn run watch --scope={gatsby,gatsby-image,gatsby-link}`
-  Watches and rebuilds the gatsby core code on any changes
+  Observa e rebuilda o código core do gatsby quando houver alguma alteração.
 - `gatsby-dev`
-  Copies over the changes from core to the example
+  Copia as alterações do core para o exemplo.
 - `gatsby develop`
-  Serves the example in development mode
+  Disponibiliza o exemplo em modo de desenvolvimento.
 
-The running example app is shown on the right in a preview window.
+A aplicação exemplo será exibida no lado direito, em uma janela de pré-visualização.
 
-## Working on an Issue
+## Trabalhando em uma Issue
 
-To start working on an issue, you can prefix the issue URL with `gitpod.io/#`.
+Para começar a trabalhar em uma issue, você pode pode prefixar a URL da issue com `gitpod.io/#`.
 
-> Example: https://gitpod.io/#https://github.com/gatsbyjs/gatsby/issues/1199
+> Exemplo: https://gitpod.io/#https://github.com/gatsbyjs/gatsby/issues/1199
 
-This will open a fresh environment with a local branch named after the issue.
-You can now code, test, commit, push and create a PR from within the workspace.
+Isso irá abrir um ambiente de desenvolvimento local zerado, com uma branch local nomeada após a issue.
```

Comment:
```suggestion
Isso irá abrir um ambiente de desenvolvimento local zerado, com uma branch local nomeada seguinho o nome da issue.
```
Não tenho certeza, mas "named after the issue" não seria algo semelhante a isso?

Reply:
concordo

Reply:
Se for copiar e colar cuidado com a palavra "seguinho", creio que seja **seguindo**. Apenas para lembrar uaheuauehueuhea 😄